src/include/cpu/power/occ.h

```diff
@@ -0,0 +1,86 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#ifndef CPU_PPC64_OCC_H
+#define CPU_PPC64_OCC_H
+
+#include <cpu/power/scom.h>
+
+#define OCC_405_SRAM_ADDRESS (0xFFF40000)
+#define OCC_OFFSET_MAIN_EP (0x6C)
+#define OCC_BRANCH_INSTR (0x4B00000200000000)
+#define BRANCH_ADDR_MASK (0x00FFFFFC)
```

Comment:
```suggestion
#define OCC_BRANCH_INSTR (0x4800000200000000)
#define BRANCH_ADDR_MASK (0x03FFFFFC)
```

https://wiki.raptorcs.com/w/images/9/9b/405_um.pdf page 199